sdlib/d/gc/extent.d

```diff
@@ -98,9 +98,12 @@ private:
 
 	Links _links;
 
+	// The largest slab slot count for which the use of flags is possible:
+	enum maxFlags = 256;
```

Comment:
This is used in one function exclusively, so it goes in the function (also, constants are capitalized).